src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -147,12 +153,12 @@ internal void CreateLuceneIndexWriter()
             {
 
                 DisposeWriter();
-                (ExecutionEvents.ActiveSession.GetParameterValue(ParameterKeys.Logger) as DynamoLogger).LogError($"LuceneNET LockObtainFailedException {ex}");
+                dynamoModel.Logger.LogError($"LuceneNET LockObtainFailedException {ex}");
```

Comment:
I had to revert this because I found in edge case if the index is ever locked, the previous code will make Dynamo hang forever